arelle/ModelXbrl.py

```diff
@@ -10,7 +10,7 @@
 from decimal import Decimal
 from arelle import UrlUtil, XmlUtil, ModelValue, XbrlConst, XmlValidate
 from arelle.FileSource import FileNamedStringIO
-from arelle.ModelDtsObject import ModelRelationship
+from arelle.ModelRelationshipSet import ModelRelationshipSet as ModelRelationshipSetClass
```

Comment:
This seems problematic. The naming conflict that this is trying to avoid is for a dynamic import from the same module (I'm not sure why it's dynamic). If that import does in fact need to be dynamic then I'd expect this to break. Otherwise the dynamic import should be dropped.

Reply:
Oh, didn't intend to cause breaking changes. I'll refactor.

Reply:
Would this work instead? It'd be good to have a defined return-type on `relationshipSet`.